data/hai/hai reveal 4 middle.txt

```diff
@@ -1047,22 +1047,22 @@ mission "Hai Reveal [A09] History of Appeasement"
 			`	When Teeneep makes a noise as if to speak, Sayari silences her with a gentle touch and thanks Xilin in the formal capacity. He briefly thanks you for delivering Batu safely and then the two excuse themselves to see if they can get a hold of someone before their day ended.`
 			choice
-				`	"Well what now?"`
+				`	"Well, what now?"`
 				`	"That's a long time."`
 			`	Teeneep sighs in exasperation. "I won't wait for your parliament to spend months deliberating. Sayari, you have a crisis of your own here, and I cannot accept a resolution without an avenue to rescue our people. I would hope you cannot either... I must go to the Unfettered. If they can be swayed then the Elders may be pressured into acting.`
 			`	"Captain <last>," she says, giving you a calculating look. "I would very much appreciate it if you'd be so kind as to take me."`
-			`	"Teeneep no! You cannot just commandeer the Captain to act as your chauffeur wherever you want to go," says Sayari. "They have an important role here, in this crisis."`
+			`	"Teeneep, no! You cannot just commandeer the Captain to act as your chauffeur wherever you want to go," says Sayari. "They have an important role here, in this crisis."`
 			# Teeneep yes.
-			`	"And what would you have them do next then?" When Sayari seems at a loss to respond Teeneep continues. "Exactly. Your crisis is mostly resolved behind closed doors, this crisis requires transportation. The thing you seem to be missing is that they're the same crisis Sayari. They can't be resolved separately."`
-			`	Sayari slumps back in her chair with a huff, for a moment looking as old as she says she is. Teeneep waits patiently while her friend spends a good thirty seconds seriously thinking about something. "She is right <last>," says Sayari eventually. "This is worth trying. Please take her to the Unfettered if that is what she wishes; their government is on <destination> but please, be careful."`
+			`	"And what would you have them do next then?" When Sayari seems at a loss to respond, Teeneep continues. "Exactly. Your crisis is mostly resolved behind closed doors, but this crisis requires transportation. The thing you seem to be missing is that they're the same crisis, Sayari. They can't be resolved separately."`
+			`	Sayari slumps back in her chair with a huff, for a moment looking as old as she says she is. Teeneep waits patiently while her friend spends a good thirty seconds seriously thinking about something. "She is right, <last>," says Sayari eventually. "This is worth trying. Please take her to the Unfettered if that is what she wishes; their government is on <destination>, but please, be careful."`
```

Comment:
It might be how they're saying it, without pausing, but it doesn't look right when it's written down and I think it ends up looking ameteurish. I would rather leave the commas in.

Reply:
I agree here, and that's why I added the commas in. ;)

But, in the end, spoken word doesn't have to follow grammatical rules, and there's precedence for breaking this specific convention in other ES conversations as well.

It is just cropping up -a lot- here in HR